### PR TITLE
Renaming blend -> sequence

### PR DESCRIPTION
**Pull request by agutenkunst**
Since sequence is more general and descriptive
the semantic is changed.

### CODE REVIEW

File: pilz_trajectory_generation/doc/structures_concept.uxf [hunk 1]
```diff
@@ -43,7 +43,7 @@ fontsize=10
       <w>190</w>
       <h>30</h>
     </coordinates>
-    <panel_attributes>GetMotionBlendRequest</panel_attributes>
```
Comment (jschleicher) [inline pilz_trajectory_generation/doc/structures_concept.uxf:46]: Please re-generate the corresponding pdf after you've changed the uxf diagram.

File: pilz_trajectory_generation/include/pilz_trajectory_generation/move_group_sequence_action.h [hunk 2]
```diff
@@ -34,32 +34,32 @@ class CommandListManager;
  * @brief Provide action to blend multiple trajectories and execute the result
```
Comment (jschleicher) [inline pilz_trajectory_generation/include/pilz_trajectory_generation/move_group_sequence_action.h]: docu?

File: pilz_trajectory_generation/test/integrationtest_command_list_manager.cpp [hunk 3]
```diff
@@ -148,8 +148,8 @@ INSTANTIATE_TEST_CASE_P(InstantiationName, IntegrationTestCommandListManager, ::
  */
```
Comment (jschleicher) [inline pilz_trajectory_generation/test/integrationtest_command_list_manager.cpp:148]: docu: `send a blending request`

File: pilz_trajectory_generation/src/move_group_sequence_action.cpp [hunk 4]
```diff
@@ -49,33 +49,33 @@
...
 {
 }
 
-void MoveGroupBlendAction::initialize()
+void MoveGroupSequenceAction::initialize()
 {
   // start the move action server
-  move_action_server_.reset( new actionlib::SimpleActionServer<pilz_msgs::MoveGroupBlendAction>(
+  move_action_server_.reset( new actionlib::SimpleActionServer<pilz_msgs::MoveGroupSequenceAction>(
                                root_node_handle_, "blend_move_group",
```
Comment (jschleicher) [inline pilz_trajectory_generation/src/move_group_sequence_action.cpp]: shouldn't we also change the action topic name?

File: pilz_trajectory_generation/test/integrationtest_sequence_action_capability.cpp [hunk 5]
```diff
@@ -60,7 +60,7 @@ class IntegrationTestBlendAction : public ::testing::Test
 
 protected:
   ros::NodeHandle ph_ {"~"};
-  actionlib::SimpleActionClient<pilz_msgs::MoveGroupBlendAction> ac_blend_{ph_, BLEND_ACTION_NAME, true};
+  actionlib::SimpleActionClient<pilz_msgs::MoveGroupSequenceAction> ac_blend_{ph_, BLEND_ACTION_NAME, true};
```
Comment (jschleicher) [inline pilz_trajectory_generation/test/integrationtest_sequence_action_capability.cpp]: shouldn't you also change the action client topic name (see comment on action server side above as well).

File: pilz_trajectory_generation/src/move_group_sequence_action.cpp [hunk 6]
```diff
@@ -49,33 +49,33 @@
 namespace pilz_trajectory_generation
 {
 
-MoveGroupBlendAction::MoveGroupBlendAction()
+MoveGroupSequenceAction::MoveGroupSequenceAction()
   : MoveGroupCapability("BlendAction")
```
Comment (jschleicher) [inline pilz_trajectory_generation/src/move_group_sequence_action.cpp]: Name of the capability?

File: pilz_trajectory_generation/src/move_group_sequence_service.cpp [hunk 7]
```diff
@@ -36,37 +36,37 @@
...
+#include "pilz_trajectory_generation/move_group_sequence_service.h"
 
 #include "pilz_trajectory_generation/capability_names.h"
 #include "pilz_trajectory_generation/command_list_manager.h"
 
 namespace pilz_trajectory_generation
 {
 
-MoveGroupBlendService::MoveGroupBlendService() : MoveGroupCapability("BlendService")
+MoveGroupSequenceService::MoveGroupSequenceService() : MoveGroupCapability("BlendService")
```
Comment (jschleicher) [inline pilz_trajectory_generation/src/move_group_sequence_service.cpp]: Capability name `BlendService` in constructor?

File: pilz_trajectory_generation/src/move_group_sequence_service.cpp [hunk 8]
```diff
@@ -36,37 +36,37 @@
...
-void MoveGroupBlendService::initialize()
+void MoveGroupSequenceService::initialize()
 {
-  blend_manager_.reset(new pilz_trajectory_generation::CommandListManager(ros::NodeHandle("~"),
+  sequence_manager_.reset(new pilz_trajectory_generation::CommandListManager(ros::NodeHandle("~"),
                                                                           context_->planning_scene_monitor_->getRobotModel()));
 
-  blend_service_ = root_node_handle_.advertiseService(BLEND_SERVICE_NAME,
-                                                      &MoveGroupBlendService::plan,
+  blend_service_ = root_node_handle_.advertiseService(SEQUENCE_SERVICE_NAME,
```
Comment (jschleicher) [inline pilz_trajectory_generation/src/move_group_sequence_service.cpp]: For the sake of completeness I'd rename this member variable `blend_service_` as well.

**Review by jschleicher — CHANGES_REQUESTED** [review]
In principal fine. See individual comments.

File: pilz_trajectory_generation/doc/structures_concept.uxf (hunk 1 above)
Comment (agutenkunst) [reply pilz_trajectory_generation/doc/structures_concept.uxf:46]: Done

File: pilz_trajectory_generation/test/integrationtest_sequence_action_capability.cpp (hunk 5 above)
Comment (agutenkunst) [reply pilz_trajectory_generation/test/integrationtest_sequence_action_capability.cpp]: Renamed.

File: pilz_trajectory_generation/src/move_group_sequence_action.cpp (hunk 6 above)
Comment (agutenkunst) [reply pilz_trajectory_generation/src/move_group_sequence_action.cpp]: Fixed.

**Review by jschleicher — APPROVED** [review]
yep. Ready to merge.

**Pull request merged**